docs/source/user/violations.rst

```diff
@@ -136,6 +136,32 @@ the errors in that file will show up without having to modify our
 configuration. Both exist so we can choose which is better for us.
 
 
+Ignoring Violations for some Files
+-----------------------
+
+Sometimes, we might want to ignore some error codes (or class of error codes) for
+some part of our project (tests for example).
+We can do this using :option:`flake8 --per-file-ignores` from CLI:
+
+.. prompt:: bash
+
+ flake8 --per-file-ignores='project/test.py:F841 setup.py:E121'
+ flake8 --per-file-ignores='tests/*:F841 setup.py:E121'
```

Comment:
```suggestion
   flake8 --per-file-ignores='project/test.py:F841 setup.py:E121'
   flake8 --per-file-ignores='tests/*:F841,D setup.py:E121'
```

Let's fix the indentation and have an example of multiple codes for one file glob.